app/services/log_ids_service.rb

```diff
@@ -0,0 +1,29 @@
+# Process a MARC file and logs the IDs deleted and updated in the file
+class LogIdsService
+  def process_file(file)
+    indexer = Traject::Indexer.new
+    indexer.load_config_file(config_file)
+    indexer.process(file)
+  end
+
+  # Note: Our MARC files have 100K records so theorically the maximum number of
+  # deleted_ids or updated_ids will be that.
+  def self.save(delete_ids:, update_ids:)
+    # Create an event...
+    ev = Event.new(start: Time.now, finish: Time.now, success: true)
```

Comment:
I'm surprised we're creating a new event and new dumps. I expected to process existing dumps and add the fields to them.

Reply:
Fixed